lib/registries/addon/drafts/draft/metadata/template.hbs

```diff
@@ -3,24 +3,27 @@
 {{else}}
     <form>
         <FormControls
-            @changeset={{changeset this.draftRegistration}}
+            @changeset={{changeset this.draftManager.metadataChangeset}}
```

Comment:
So we can also pass a changeset this helper, can't we just pass in the changeset?

Reply:
I'm not too familiar with how the `changeset` helper works, but when I tried to pass in just `this.draftManager.metadataChangeset`, it did not seem to work

Reply:
🤔 that maybe another issue worth looking into. Because, according to docs, the helper is used to create a new changeset based on the object you pass it and any validator functions.

Reply:
Good eye, @fabmiz! This should be just:
```suggestion
            @changeset={{this.draftManager.metadataChangeset}}
```

Reply:
I was doing `@changeset=this.draftManager.metadataChangeset` (no {{ }} 🤦‍♂ )